### PR TITLE
Added french translation for fuel core lang files

### PR DESCRIPTION
**Pull request by Seb-C**

### CODE REVIEW

**Comment by smeeckaert** [comment]
:+1:

File: lang/fr/validation.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+return array(
+	'required'        => 'Le champ :label est requis et doit contenir une valeur.',
+	'min_length'      => 'Le champ :label doit contenir au moins :param:1 caractères.',
+	'max_length'      => 'Le champ :label ne doit pas contenir plus de :param:1 caractères.',
+	'exact_length'    => 'Le champ :label doit contenir exactement :param:1 caractères.',
+	'match_value'     => 'Le champ :label doit contenir la valeur :param:1.',
+	'match_pattern'   => 'Le champ :label doit correspondre au motif :param:1.',
+	'match_field'     => 'Le champ :label doit correspondre au champ :param:1.',
+	'valid_email'     => 'Le champ :label doit contenir un email valide.',
+	'valid_emails'    => 'Le champ :label doit contenir une liste d\'emails valides.',
```
Comment (Meroje) [inline lang/fr/validation.php]: Here, have a [smart quote](http://smartquotesforsmartpeople.com/): `’`

**Comment by tonyyb** [comment]
:+1:

**Comment by Meroje** [comment]
:+1:

**Comment by shaoshiva** [comment]
:+1:

**Pull request merged**